server/src/http_server/pages/blog/md/html.rs

```diff
@@ -19,6 +21,7 @@ use crate::AppConfig;
 pub struct SyntaxHighlightingContext {
     pub(crate) theme: syntect::highlighting::Theme,
     pub(crate) syntax_set: syntect::parsing::SyntaxSet,
+    pub(crate) current_article_path: Option<String>,
```

Comment:
We can remove this now that we have the new `MarkdownRenderContext` struct right?

Reply:
Did this come back in a merge maybe?

We have the `MarkdownRenderContext` so I think we can/want to remove this path from this struct!